sdk/js/src/index.ts

```diff
@@ -379,10 +380,12 @@ export class Client {
         encoded_order,
         Order.discriminator.length
       );
-      const remainingOutputAmount =
+      const remainingOutputAmount = BN.max(
```

Comment:
I prefer using `anchor.BN` to remain consistent with the rest of the file or change everything to this new format.
The main reason is to explicitly mention this is the BN exposed by anchor and not the BN library itself.

Reply:
fixed